### PR TITLE
pin pytest-cov==2.6.0

### PR DESCRIPTION
**Pull request by singuliere**
Fixes: https://github.com/ansible/molecule/issues/1657

#### PR Type

- Bugfix Pull Request

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -2,7 +2,8 @@ mock
 pur
 # pytest==3.1.0 issues with pytest-verbose-parametrize
 pytest==3.0.7
-pytest-cov==2.6.0
+pytest-cov<2.6.1; python_version < "3.6"
```
Comment (singuliere) [inline test-requirements.txt]: the changed requirement is for [pytest>=3.6](https://github.com/pytest-dev/pytest-cov/compare/v2.6.0...master#diff-2eeaed663bd0d25b7e608891384b7298R124) and not python 3.6

**Comment by singuliere** [comment]
Re-pushed after checking that [travis CI](https://travis-ci.com/ansible/molecule/builds/96588015) fails with https://github.com/ansible/molecule/pull/1658/commits/966e9f39f1a01cc21d4656a30f2f7b768cd2e1c9

File: test-requirements.txt (hunk 1 above)
Comment (webknjaz) [reply test-requirements.txt]: So it should be pinned with markers too

**Comment by singuliere** [comment]
Two [tests fail](https://travis-ci.com/ansible/molecule/jobs/168882598) but there are no error messages:

```
----------------------------- Captured stderr call -----------------------------
ERROR:
--------------------------- Captured stdout teardown ---------------------------
```

**Comment by singuliere** [comment]
@webknjaz given the current instability of Travis and the fact that most tests pass, would you consider merging this pull request? Without it everything fails so the situation is worse.

**Comment by singuliere** [comment]
@webknjaz thanks for the close/open pull request trick to trigger the CI, much more convenient that repushing. Fingers crossed that travis will be faster in the morning :crossed_fingers:

**Comment by singuliere** [comment]
The tests all pass based on the **two** following Travis CI runs

* https://travis-ci.com/ansible/molecule/builds/96645217
* https://travis-ci.com/ansible/molecule/builds/96714750

all tests pass. A [single job](https://travis-ci.com/ansible/molecule/jobs/169132028) fails in [this run](https://travis-ci.com/ansible/molecule/builds/96714750) but the [same job succeeds](https://travis-ci.com/ansible/molecule/jobs/168882600) in [the other run](https://travis-ci.com/ansible/molecule/builds/96645217).

**Comment by webknjaz** [comment]
I've done some restarts, let's see how it goes. It looks like we'll need to split more jobs after all...

File: test-requirements.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@ mock
 pur
 # pytest==3.1.0 issues with pytest-verbose-parametrize
 pytest==3.0.7
-pytest-cov
+pytest-cov==2.6.0
```
Comment (webknjaz) [inline test-requirements.txt]: I still want conditionals I think.

Comment (webknjaz) [reply test-requirements.txt]: env markers. anyway, let's merge it as is for now. but for future I'd like to upgrade pytest, there's even a new major version out there.

Comment (singuliere) [reply test-requirements.txt]: I'm not sure what env marker would be relevant in that context, hence my question. There is no dependency on a specific version of python and I can't think of any other env marker.

**Comment by singuliere** [comment]
thanks for the merge :-)

**Pull request merged**